rfcs/044-browse-badges.md

```diff
@@ -0,0 +1,61 @@
+- Feature Name: Badges on landing page & browse view
+- Start Date: 2021-09-28
+- RFC PR: [amundsen-io/rfcs#44](https://github.com/amundsen-io/rfcs/pull/44)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Badges on landing page & browse view
+
+## Summary
+
+This RFC proposes to introduce `Popular/All badges` section below `Popular tags` on landing page and & browse section.*
+
+! `Popular/All` is either this or that choice that has been explained in `Unresolved questions` section.
+
+## Motivation
+
+After introducing refactor of tags and badges only the former ones get dedicated section on landing page and . As badges
```

Comment:
could you link to what you're mentioning. I'm out of the loop.

Reply:
This referred to some Slack threads that were removed due to retention period. Which part you're interested in?